.gitignore

```diff
@@ -1,2 +1,3 @@
 *.log
 .vagrant/
+.idea/
```

Comment:
In general this should go in `~/.config/git/ignore`, since it's specific to the particular tooling you are using.  The `.gitignore` in the repo is for files that are created by the tooling used by _everyone_ using the repo.  (No need to change anything here, since it does not matter for this repo – just a general hint.)

Reply:
Ah you're right :) For me it's actually in `~/.gitignore`, but either way it does contain `.idea/` already.